src/providers/sysinfo.rs

```diff
@@ -110,19 +137,13 @@ impl Sysinfo {
             }
 
             self.render_stat(3, &mut buffer, format!("{}: {:>4}{}", net_direction, adjusted_net_load, net_load_unit), net_load / (self.net_load_max * 1024_f64.pow(2)))
-        }).unwrap_or_else(|| {
-            warn!("couldn't find net interface `{}`", self.net_interface_name);
-            Ok(())
-        })?;
+        });
 
         self.sys.components().iter().find(|component|
             component.label() == self.sensor_name
         ).map(|c| {
             self.render_stat(4, &mut buffer, format!("T: {:>4.1}C", c.temperature()), c.temperature() as f64 / self.temperature_max)
-        }).unwrap_or_else(|| {
-            warn!("couldn't find sensor `{}`", self.sensor_name);
-            Ok(())
-        })?;
+        });
```

Comment:
This result is unused. That could lead to it having no effect. Can you fix that? You can use `cargo clippy` to find mistakes like these.

Reply:
Yes I will try to fix that, it's my first time in rust so there's a lot that I have to learn